loop_live.py

```
@@ -0,0 +1,644 @@
+                        elif message.get("type") == "tool_use":
+                            display_output = f"Calling tool: {message.get('name', '')}"
+                            display_output += f"Input: {json.dumps(message.get('input', {}))}"
+                            display.add_message("user", "tool use")
+                        else:
+                            # display.add_message("user", "First Else")
+                            if len(messages) == 1:
+                                display_output = message['content']
+                            else:
+                                try:
+                                    display_output = message['content'][0]['content'][0]['text']
+                                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)